styles/Debug.module.css

```diff
@@ -0,0 +1,16 @@
+.overlay {
+  position: absolute;
```

Comment:
Maybe add some styling to prevent overflow.

```css
max-height: 100vh;
overflow: auto;
```

Reply:
just did that :)